File: nature-harvest-frontend/src/app/dtos/payment/vnpay-payment.dto.ts
import { OrderDto } from '../order/order.dto';

export class VnPayPaymentDto {
  bankCode: string;
  language: string;
  order: OrderDto;

  constructor(data: any) {
    this.bankCode = data.bankCode;
    this.language = data.language;
    this.order = data.order;
  }
}

File: nature-harvest-backend/nature-harvest-backend/src/main/java/com/api/nature_harvest_backend/dtos/payment/VnPayPaymentDto.java
package com.api.nature_harvest_backend.dtos.payment;

import com.api.nature_harvest_backend.dtos.order.OrderDto;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class VnPayPaymentDto {
    private String bankCode;
    private String language;
    private OrderDto order;
}
